cfgov/templates/welcome.html

```diff
@@ -3,14 +3,42 @@
 
 {% block furniture %}
     <div class="content-wrapper">
+
         <h1>{% trans 'Welcome' %} <br/>
-        You've logged in to the consumerfinance.gov editorial tools.	
+        {% if 'build' in request.site.hostname %}
+            You've logged in to the demo Wagtail.    
+        {% else %}
+            You've logged in to the consumerfinance.gov editorial tools.    
+        {% endif %}
         </h1>
-        <h2>Continue on to...</h2>
+
+        <h2>Choose which admin interface you'd like to go to</h2>
         <ul>
-        {% for name, url in destinations %}
-            <li><a href="{{url}}"> {{name}}</a></li>
-        {% endfor %}
+            {% for name, url in destinations %}
+                <li><a href="{{url}}"> {{name}}</a></li>
+            {% endfor %}
         </ul>
+
+        {% if 'build' in request.site.hostname %}
+            <h2>Helpful links</h2>
+            <ul>
+                <li>
+                    <a href="https://github.cfpb.gov/pages/flapjack/wagtail-documentation/">
```

Comment:
Um, no I am not comfortable making the Wagtail documentation public at this moment given the purpose and content.

It's an interesting idea in the long run but not something we can just do right now. I don't know how to get around the GHE link. Perhaps we can't include that link here if it means putting GHE in the public.